lib/cdo/contact_rollups/v2/pardot.rb

```diff
@@ -22,41 +22,45 @@ def initialize
     @updated_prospect_deltas = []
   end
 
-  # Retrieves new email-id mappings from Pardot
-  # @param [Integer] last_id
-  # @return [Array] an array of hash {email, id}
+  # Retrieves new (email, Pardot ID) mappings from Pardot
+  #
+  # @yieldreturn [Array<Hash>] an array of hash {email, id}
```

Comment:
typo in comment?

Reply:
Do you mean `yieldreturn`? It is this tag https://rubydoc.info/gems/yard/file/docs/Tags.md#yieldreturn.

Reply:
yep nvm I assumed there should be a space or something